share/completions/git.fish

```diff
@@ -892,7 +892,7 @@ complete -f -c git -n '__fish_git_using_command filter-branch' -l subdirectory-f
 complete -f -c git -n '__fish_git_using_command filter-branch' -l prune-empty -d 'Ignore empty commits generated by filters'
 complete -f -c git -n '__fish_git_using_command filter-branch' -l original -d 'Use this option to set the namespace where the original commits will be stored'
 complete -r -c git -n '__fish_git_using_command filter-branch' -s d -d 'Use this option to set the path to the temporary directory used for rewriting'
-complete -c git -n '__fish_git_using_command filter-branch' -s f -l force -d 'Force filter branch to start even w/ refs in refs/original or existing temp directory'
+complete -c git -n '__fish_git_using_command filter-branch' -s f -l force -d 'Force filter branch to start even w/ refs in refs/original or existing temp dir'
```

Comment:
This is one of those descriptions where we're so close to the git documentation I have no actual idea what it means.

It's possible this is talking about an unclean shutdown kind of situation? Does filter-branch use the refs/original directory to store the branches before filtering, and a temp dir to work in, and so if those exist that indicates that you tried to filter-branch and didn't complete it?

But as it stands the "Force filter branch to start" is the part that wastes characters, so how about

    Filter even with refs in refs/original or existing temp directory

That's 65 characters compared to the original 85 and your 79, and it even ditches the awkward "w/".